### PR TITLE
Restructure Block and utils to match the tech proposal - Closes #4207

### LINKED ISSUE #4207: Refactor the Block domain to match the proposed architecture
### Description

As a part of the Make Lisk Desktop blockchain agnostic, we should create a domain for Block. Said domain is responsible for collecting, displaying blocks on the network.

### Acceptance Criteria

According to the TECH proposal document, this domain should have components to:
- List all blocks on the network with custom filters.
   - Apply custom filters as currently exists.
   - Apply sorting as currently exists.
- Display details of a single block.
- Display block details.

### Additional Information
Listing transactions of a block is handled by the Transaction domain, and the blocks view in the View domain is responsible for composing the compound view.

### PR DESCRIPTION
**Pull request by gadjacobs**
### What was the problem?
This PR resolves #4207

### How was it solved?
- move blocks file to own directory

### How was it tested?
end to end test will resolve with https://github.com/LiskHQ/lisk-desktop/issues/4263

### CODE REVIEW

File: packages/block/constants/constants.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+export const httpPrefix = '/api/v2';
```
Comment (soroushm) [inline packages/block/constants/constants.js]: constants file should name it as const
https://www.freecodecamp.org/news/when-to-capitalize-your-javascript-constants-4fabc0a4a4c4/ 

also blocksFilters is not a const its more like a map please rename it to "blocksFiltersMap" and move it to "map/blocksFiltersMap" under the domain


wsMethods
httpPaths
are config please create a config.js and move them there

File: packages/block/utilities/api/getBlockProps.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export const getBlockProps = ({ blockId, height }) => {
```
Comment (soroushm) [inline packages/block/utilities/api/getBlockProps.js]: please rename directory to utils
also for this api file please create a directore called api and move them there

File: packages/block/utilities/api/index.js [hunk 3]
```diff
@@ -1,140 +1,5 @@
...
- * @param {Object} network - Redux network state
- */
-export const blockUnsubscribe = () => {
-  unsubscribe(wsMethods.blocksChange);
-};
+export { getBlock } from './getBlock';
+export { getBlocks } from './getBlocks';
+export { getBlockProps } from './getBlockProps';
+export { blockUnsubscribe } from './blockUnsubscribe';
+export { blockSubscribe } from './blockSubscribe';
```
Comment (soroushm) [inline packages/block/utilities/api/index.js]:
```suggestion
export * from './blockUnsubscribe';
export * from './blockSubscribe';
```

**Review by soroushm — COMMENTED** [review]
init

**Review by soroushm — DISMISSED** [review]

File: src/modules/block/config/wsMethods.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+export default {
+    blocksChange: 'update.block',
+  };
```
Comment (ikem-legend) [inline src/modules/block/config/wsMethods.js]: @gadjacobs Please fix end of file error

**Review by ikem-legend — APPROVED** [review]

**Review by soroushm — APPROVED** [review]
gladly approved

**Pull request merged**